packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -38,20 +38,12 @@ contract Multisig is PluginUUPSUpgradeable, ProposalUpgradeable, Addresslist {
     /// @param startDate The timestamp when the proposal starts.
     /// @param endDate The timestamp when the proposal expires.
     struct ProposalParameters {
-        uint256 minApprovals;
+        uint16 minApprovals;
```

Comment:
Why `uint16 minApprovals;` while `uint32 approvals;`? 
Doesn't this causes unnecessary type conversion when comparing the two?

Reply:
not relevant anymore. already fixed.